app/controllers/analytics/districts_controller.rb

```diff
@@ -15,39 +15,40 @@ def show
       format.html
       format.csv do
         set_facility_keys
-        send_data render_to_string('show.csv.erb'), filename: download_filename
+        send_data render_to_string("show.csv.erb"), filename: download_filename
       end
     end
   end
 
   def share_anonymized_data
     recipient_email = current_admin.email
-    recipient_name = recipient_email.split('@').first
+    recipient_name = recipient_email.split("@").first
 
     AnonymizedDataDownloadJob.perform_later(recipient_name,
-                                            recipient_email,
-                                            { district_name: @organization_district.district_name,
-                                              organization_id: @organization_district.organization.id },
-                                            'district')
+      recipient_email,
+      {district_name: @organization_district.district_name,
+       organization_id: @organization_district.organization.id},
+      "district")
 
     redirect_to analytics_organization_district_path(id: @organization_district.district_name),
-                notice: I18n.t('anonymized_data_download_email.district_notice',
-                               district_name: @organization_district.district_name)
+      notice: I18n.t("anonymized_data_download_email.district_notice",
+        district_name: @organization_district.district_name)
   end
 
   def patient_list
     recipient_email = current_admin.email
 
-    PatientListDownloadJob.perform_later(recipient_email, 'district', {
+    PatientListDownloadJob.perform_later(recipient_email, "district", {
       district_name: @organization_district.district_name,
       organization_id: @organization_district.organization.id
     })
 
     redirect_to(
       analytics_organization_district_path(id: @organization_district.district_name),
-      notice: I18n.t('patient_list_email.notice',
-                     model_type: "district",
-                     model_name: @organization_district.district_name))
+      notice: I18n.t("patient_list_email.notice",
+        model_type: "district",
+        model_name: @organization_district.district_name)
+    )
```

Comment:
Related to my hash formatting comment, this inconsistency makes me raise an eyebrow as well. The `redirect_to` lists its arguments on new lines, but the `I18n` includes its first argument on the same line as the method. Is this just an artefact of the autocorrect, or does standardrb have rules explaining this behavior? I'd love to nail this down, as I think it's one of the more problematic style areas.

Here's what I'd love to see:

```ruby
redirect_to(
  analytics_organization_district_path(id: @organization_district.district_name),
  notice: I18n.t(
    "patient_list_email.notice",
    model_type: "district",
    model_name: @organization_district.district_name
  )
)
```

allowing, of course, one-lining the entire method if it isn't too long.